module3-crud-and-data-models/r1-schema-and-data-models/README.md

```diff
@@ -0,0 +1,161 @@
+## Intoduction to schemas and data modeling
+
+### Data modeling
+
+A database model is a type of data model that determines the logical structure of a database. It is the high level design which defines the kind of tables, the `fields` in those tables and the `relations` between different tables.
```

Comment:
you shouldnt be using `code` for non-code terms, it's not really a thing. bold it if it's a term. and also, if you are assuming they know what fields and relations are, you dont need to bold them (it's explained in module 2, and also you didn't write any definitions)